definitions/fake-ec-validation/validate_enterprise_contract.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: validate-enterprise-contract
+spec:
+  params:
+    - name: applicationSnapshot
+      type: string
+      description: >
+        JSON representation of the ApplicationSnapshot being released
+      default: ""
+    - name: policy
+      type: string
+      description:
```

Comment:
Why is line 11 `description: >` and this one `description:` when both are followed by single line strings?

Reply:
`>` allows you to have multiline strings. I do it in line 11 because it's too long to pass yaml validation. Line is short enough after a single line break.

Reply:
I don't follow. Neither line 11 nor 16 are multi line strings

Reply:
What I was trying to say by "I don't follow" is that your previous explanation did not make sense to me